### PR TITLE
UI improved

### PR DESCRIPTION
**Pull request by Frahane**
Screenshot_20230905-190008
Screenshot_20230905-190824
Screenshot_20230907-133803
Screenshot_20230907-140146

### CODE REVIEW

File: web/exchange-form-slot.php [hunk 1]
```diff
@@ -1,65 +1,148 @@
...
+// Check if the slot is empty (not found)
 if (empty($slot)) {
+    // If empty, send a 404 Not Found header and exit with an error message
     header($_SERVER["SERVER_PROTOCOL"] . " 404 Not Found");
     die('Slot not found');
 }
 
+// Set the status based on the slot's status
 $status = strtolower($slot['status']);
-
```
Comment (Alex014) [inline web/exchange-form-slot.php:26]: We need this part

**Review by Alex014 — CHANGES_REQUESTED** [review]

File: web/exchange-form-slot.php (hunk 1 above)
Comment (Alex014) [reply web/exchange-form-slot.php:26]: (slot processing)

Comment (Frahane) [reply web/exchange-form-slot.php:26]: Understood! And sorry for the delay.

Comment (Frahane) [reply web/exchange-form-slot.php:26]: Done. It was removed by mistake. Sorry.
Please, let me know if there's any other. ☺️

**Review by Alex014 — APPROVED** [review]
Ok on a first look
Needs to be tested

**Pull request merged**